cirq-core/cirq/experiments/t1_decay_experiment_test.py

```diff
@@ -121,7 +121,7 @@ def test_all_on_results():
         data=pd.DataFrame(
```

Comment:
can you change this line to 

```
desired = ...
assert results == desired, f'{results.data=} {desired.data=}'
```

so that we get a more informative error message.

Reply:
I tried this, and it just outputs
```
>       assert results == desired, f'{results.data=} {desired.data=}'
E       AssertionError: results.data=   delay_ns  false_count  true_count
E         0       100           10           0
E         1       215           10           0
E         2       464           10           0
E         3      1000           10           0 desired.data=   delay_ns  false_count  true_count
E         0       100           10           0
E         1       215           10           0
E         2       464           10           0
E         3      1000           10           0
```

Reply:
I suspect it may have something to do with data types since I changed line 83 in `t1_decay_experiment.py` to `np.logspace(np.log10(min_delay_nanos), np.log10(max_delay_nanos), num_points, dtype=int)`.

Reply:
looks like it's indeed a types issue specifically for the `delay_ns` column
```py3
>>> results.data.dtypes
delay_ns       float64
false_count      int64
true_count       int64
dtype: object
>>> desired.data.dtypes
delay_ns       int64
false_count    int64
true_count     int64
dtype: object
```

Reply:
That's pretty strange. When I run it on linux, `delay_ns` is `int64`, and in line 83 of `t1_decay_experiment.py`, I specifically ask for `dtype=int`. What do you recommend? Should I manually convert `delay_ns` to integers in the test? It seems like there is some deeper issue when running on windows.

Reply:
the result is computed by `sampler.sample` which is probably changing the dtype to float internally. I suggest that you ensure that the `desired` result is also float.

Why do you want to force `delay_ns` to be `int`?

Reply:
I changed it back to `float`